app/femr/business/services/system/UpdatesService.java

```diff
@@ -0,0 +1,160 @@
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends INetworkStatus>> retrieveNetworkStatuses() {
+        ServiceResponse<List<? extends INetworkStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends INetworkStatus> networkStatuses = networkStatusRepository.findAll(NetworkStatus.class);
+            response.setResponseObject(networkStatuses);
+
+        } catch (Exception ex) {
+            response.addError("", ex.getMessage());
+        }
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends INetworkStatus>> updateNetworkStatuses() {
+        ServiceResponse<List<? extends INetworkStatus>> response = new ServiceResponse<>();
+        //TODO: Lemur Team update the database with the right values
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IKitStatus>> retrieveKitStatuses() {
+        ServiceResponse<List<? extends IKitStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends IKitStatus> kitStatuses = kitStatusRepository.findAll(KitStatus.class);
+            response.setResponseObject(kitStatuses);
+
+        } catch (Exception ex) {
+            response.addError("", ex.getMessage());
+        }
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IKitStatus>> updateKitStatuses() {
+        ServiceResponse<List<? extends IKitStatus>> response = new ServiceResponse<>();
+        //TODO: Lemur Team update the database with the right values
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IDatabaseStatus>> retrieveDatabaseStatuses() {
+        ServiceResponse<List<? extends IDatabaseStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends IDatabaseStatus> kitStatuses = databaseStatusRepository.findAll(DatabaseStatus.class);
+            response.setResponseObject(kitStatuses);
+
+        } catch (Exception ex) {
+            response.addError("", ex.getMessage());
+        }
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IDatabaseStatus>> updateDatabaseStatuses() {
+        ServiceResponse<List<? extends IDatabaseStatus>> response = new ServiceResponse<>();
+        //TODO: Feather Team backup database ---> run script to see if it successsfully return if it succeeds or not
+
+        String updated_date = java.time.LocalDate.now().toString().replace("-", ".");
+        DatabaseStatus databaseStatus = new DatabaseStatus();
+        databaseStatus.setName("Last Backup");
+        databaseStatus.setValue(updated_date);
+        databaseStatusRepository.update(databaseStatus);
+
```

Comment:
Do we still need this code?

Reply:
I think adding the date in two places was a backup thing?